### ISSUE TITLE
Incorrect parsing due to + operator

### ISSUE THREAD

**Issue by ashtonG** [mattn/emmet-vim#88]
When writing the expression:

```
.container+#parent>(#firstChild>(label+input[type=text])*2)+(#secondChild>input[type=text]*3)
```

zencoding-vim expands to 

``` html
<div class="container"></div>
<div id="parent">
    <div id="firstChild">
        <label for=""></label>
        <input type="text" />
        <label for=""></label>
        <input type="text" />
        <div id="secondChild"><input type="text" /><input type="text" /><input type="text" /></div>
    </div>
</div>
```

When it should expand to:

``` html
<div class="container"></div>
<div id="parent">
  <div id="firstChild">
    <label for=""></label>
    <input type="text" />
    <label for=""></label>
    <input type="text" />
  </div>
  <div id="secondChild">
    <input type="text" />
    <input type="text" />
    <input type="text" />
  </div>
</div>
```

The latter was generated using the codemirror parser.

I've isolated it to the `+` operator in the `(label+input[type=text])*2` part of the expression. Replacing it with a `>` leads to the expected turnout:

given:

```
.container+#parent>(#firstChild>(label>input[type=text])*2)+(#secondChild>input[type=text]*3)
```

we get:

``` html
<div class="container"></div>
<div id="parent">
    <div id="firstChild">
        <label for=""><input type="text" /></label>
        <label for=""><input type="text" /></label>
    </div>
    <div id="secondChild"><input type="text" /><input type="text" /><input type="text" /></div>
</div>
```

Which, save for the lack of line breaks in the secondChild `div` element is the same output from the codemirror parser. 

In addition, trying either expression without the `.container` element results in:

``` html
<div id="parent">
</div>
</div>
```

In other words, there needs to be a + operation on the parent element or the parsing ignores the contents of the parentheses.

**EDIT: Forgot to change expression**
